internal/cmd/gocdk/main.go

```diff
@@ -81,12 +84,12 @@ type processContext struct {
 	workdir string
 	env     []string
 	stdin   io.Reader
-	stdout  io.Writer
-	stderr  io.Writer
+	outlog  *log.Logger
+	errlog  *log.Logger
 }
 
-// osProcessContext returns the default process context from global variables.
-func osProcessContext() (*processContext, error) {
+// newDefaultProcessContext returns the default processContext from global variables.
+func newDefaultProcessContext() (*processContext, error) {
```

Comment:
Why the change from `osProcessContext`? It uses the operating system process handles, so the name seems fitting.

Reply:
OK, done.